docs/guide/homeassistant.md

```diff
@@ -208,3 +208,105 @@ The main template is like the thermostat template. The things to add are:
   }
 }
 ```
+# Adding Devices Manually
+
+Some people prefer to not use discovery and so here are some examples of devices you can use:
+
+Note the settings used here for the Gateway in the UI are:
+Type: ValueID topics
+Payload type: Just Value
+Use node name instead of numeric nodeIDs: true
+Ignore location: true
+
+::: Motion Sensor
```

Comment:
What is this `:::` ? I think you need to use a `###`
```suggestion
### Motion Sensor
```

Reply:
these i replicated from room-assistant.io

It was the expandable chunks i mentioned on slack - example here: 
https://www.room-assistant.io/guide/configuration.html#configuring-with-docker

Reply:
not sure if thats the correct markdown for it or not?